cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1912,6 +2006,7 @@ action.cyences_notable_event_action.param.filter_macro_name = cs_o365_failed_log
 action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity"  Workload=AzureActiveDirectory Operation=UserLoginFailed LogonError="BlockedByConditionalAccess"
 action.cyences_notable_event_action.system_compromised_search = | stats count by user
 action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365` sourcetype="o365:management:activity"  Workload=AzureActiveDirectory Operation=UserLoginFailed LogonError="BlockedByConditionalAccess" user=$row.user$
+action.cyences_send_email_action = 1
 
 [O365 - Daily Login Failure]
```

Comment:
This is a kind of daily report. I think it should be included on the overview page. I'm not sure 100%

Reply:
It's already included in the overview page.